Tutorials/WinForms/Classic/create-persistent-classes-and-connect-xpo-to-database.md

```diff
@@ -0,0 +1,99 @@
+# Create persistent classes and connect XPO to the database
+
+&nbsp;&nbsp;&nbsp;
+[Back to TOC](../../)
+&nbsp;&nbsp;&nbsp;
+[Step 2](/connect-data-grid-to-xpo-objects.md)   
+
+* Create a new Windows Forms App project in Visual Studio.
+* Add references to these assemblies ([Manage references in a project](https://docs.microsoft.com/en-us/visualstudio/ide/managing-references-in-a-project)):  
+  **DevExpress.Data**  
+  **DevExpress.Xpo**  
+  These assemblies are available under the **Assemblies > Extensions** category in the **Reference Manager**. 
+* Create the **DataAccess** folder and add the `Customer` class to it. This class is a model for the Customer table in a database.
+* Change the base class to `XPObject` and add a constructor with a `session` parameter:  
+    ``` csharp
+    using DevExpress.Xpo;
+
+    namespace DxSample.DataAccess {
+        public class Customer :XPObject {
+            public Customer(Session session) : base(session) { }
+        }
+    }
+    ```
+    **See also:**   
+    [XPO Classes Comparison](https://docs.devexpress.com/XPO/3311/concepts/xpo-classes-comparison)
+* Add the `FirstName` and `LastName` properties to the `Customer` class. XPO maps these proeprties to columns in the Customer table.
```

Comment:
"proeprties"? You should proofread your text.

Reply:
I proofread all the files: a496223cfeb605e156bdbe699be41915819cbb9c